src/main/java/com/stripe/model/PersonCollection.java

```diff
@@ -0,0 +1,63 @@
+package com.stripe.model;
+
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.Map;
+
+public class PersonCollection extends StripeCollection<Person> {
+  // <editor-fold desc="all">
+  @Deprecated
+  public PersonCollection all(Map<String, Object> params) throws StripeException {
```

Comment:
`all()` methods are deprecated. You've added `list()` methods below, no need to have `all()` methods too. We removed most of these in the big v6 overhaul.

Reply:
Did we miss some? If not why are they still here? I mostly copy/pasted another resource when I added this

Reply:
`BitcoinReceiverCollection`, `BitcoinTransactionCollection` and `ExternalAccountCollection` still have `all()` methods. We should probably remove all of those in the next major version.